editor/src/components/canvas/canvas-strategies/absolute-move-strategy.ts

```diff
@@ -38,27 +45,58 @@ export const absoluteMoveStrategy: CanvasStrategy = {
       interactionState.interactionData.drag != null
     ) {
       const drag = interactionState.interactionData.drag
-      return canvasState.selectedElements.flatMap((selectedElement) => [
-        adjustNumberProperty(
-          'permanent',
-          selectedElement,
-          stylePropPathMappingFn('left', ['style']),
-          drag.x,
-          true,
-        ),
-        adjustNumberProperty(
-          'permanent',
-          selectedElement,
-          stylePropPathMappingFn('top', ['style']),
-          drag.y,
-          true,
-        ),
+      return [
+        ...canvasState.selectedElements.flatMap((selectedElement) => {
+          const element: JSXElement | null = withUnderlyingTarget(
+            selectedElement,
+            canvasState.projectContents,
+            {},
+            canvasState.openFile,
+            null,
+            (_, e) => e,
+          )
+          const elementParentBounds =
+            MetadataUtils.findElementByElementPath(
+              sessionState.startingMetadata, // TODO should this be using the current metadata?
```

Comment:
I think that would only make sense if this strategy could affect the parent bounds, _but_ this code is being called by the reparenting strategy :grimacing:

Reply:
good shout, I'm gonna make a todo to figure out what do we want to do if the move affects the parent bounds

Reply:
todo made